ja/ruby/standard.md

```diff
@@ -273,6 +273,14 @@ end
 
 # good
 do_something if some_condition
+
+# bad
+foo = "This is a very very long string that can not be broken down and may contain #{variable}" unless some_condition
+
+# good
+unless some_condition
+  foo = "This is a very very long string that can not be broken down and may contain #{variable}"
```

Comment:
@johnny-miyake @norifumi 
This line is longer than 80 characters itself, should we break it to 2 line?
```ruby
foo = "This is a very very long string that can not be broken down \
   and may contain #{variable}"
``